src/warplib/implementations/external_input_checks/externalInputChecksInts.ts

```diff
@@ -5,31 +5,19 @@ const INDENT = ' '.repeat(4);
 export function external_input_check_ints(): WarplibFunctionInfo {
   return {
     fileName: 'external_input_check_ints',
-    imports: [
-      'from starkware.cairo.common.math_cmp import is_le_felt',
-      'from starkware.cairo.common.uint256 import Uint256',
-    ],
+    imports: [],
     functions: forAllWidths((int_width) => {
       if (int_width === 256) {
         return [
-          `func warp_external_input_check_int256{range_check_ptr}(x : Uint256){`,
-          `${INDENT}alloc_locals;`,
-          `${INDENT}let inRangeHigh : felt = is_le_felt(x.high, ${mask(128)});`,
-          `${INDENT}let inRangeLow : felt = is_le_felt(x.low, ${mask(128)});`,
-          `${INDENT}with_attr error_message("Error: value out-of-bounds. Values passed to high and low members of Uint256 must be less than 2**128."){`,
-          `${INDENT.repeat(2)}assert 1 = (inRangeHigh * inRangeLow);`,
-          `${INDENT}}`,
-          `${INDENT}return();`,
+          `fn warp_external_input_check_int256(x : u256) {`,
+          `${INDENT}return ();`,
```

Comment:
This function has no use, no need to generate it

Reply:
It's just to keep compatibility with all the code that use this function. At the time this PR was created the Cairo func gen was not merged yet so exist a lot of Cairo code written by it that could not be updated to avoid future merge conflicts

Reply:
I think the best solution is just not to generate it and then go to the parts of the project which requires it and erase them

Reply:
Ok